test/train-sets/ref/houses.stdout

```diff
@@ -1,30 +1,30 @@
 0
-HiddenBias:29015:1:0@0
+	HiddenBias:29015:1:0@0
```

Comment:
@jackgerrits what triggered this change? I feel this was from an old change of formatting you did a while back